app/models/course.rb

```diff
@@ -540,7 +541,7 @@ def gcse_grade_required
   end
 
   def last_published_at
-    latest_enrichment&.last_published_timestamp_utc
+    current_published_enrichment&.last_published_timestamp_utc || enrichment_attribute(:last_published_timestamp_utc)
```

Comment:
any reason this couldn't be the max `last_published_timestamp_utc` from all associated enrichments?